README.md

```diff
@@ -1,7 +1,137 @@
+![Giraffe](https://raw.githubusercontent.com/giraffe-fsharp/Giraffe/master/giraffe.png)
+
 # Giraffe.OpenApi
 
-> [!IMPORTANT]
-> This is currently only works with my branch of Giraffe, which can be found [here](https://github.com/mrtz-j/Giraffe/tree/configureEndpoint).
+An extension for the [Giraffe](https://github.com/giraffe-fsharp/Giraffe) Web Application framework with functionality to auto generate OpenApi documentation spec from code.
+
+[![NuGet Info](https://buildstats.info/nuget/Giraffe.OpenApi?includePreReleases=true)](https://www.nuget.org/packages/Giraffe.OpenApi/)
+
+
+## Table of Contents 
+
+- [About](#about)
+- [Getting Started](#getting-started)
+- [Documentation](#documentation)
+  - [Integration](#integration)
+  - [addOpenApi](#addopenapi)
+  - [addOpenApiSimple](#addopenapisimple)
+  - [configureEndpoint](#configureendpoint)
+- [License](#license)
+
+## About
+
+`Giraffe.OpenApi` is a library that extends the `Giraffe` Web Application framework with functionality to auto generate OpenApi documentation spec from code. This means that you can define your API endpoints using Giraffe and generate OpenApi or Swagger documentation from it.
+
+Inspired by the [Oxpecker.OpenApi](https://github.com/Lanayx/Oxpecker/blob/develop/src/Oxpecker.OpenApi) library, but adapted to work with Giraffe.
+
+## Getting Started
+
+Add the `Giraffe.OpenApi` NuGet package to your project:
+
+```bash
+dotnet add package Giraffe.OpenApi
```

Comment:
The PR looks great.

Is there any chance to merge this and publish a Nuget package version of `Giraffe.OpenApi`?

Reply:
Sure, I'm just waiting for another maintainer to approve the PR

Reply:
@dbrattli @nojaf Can you please help?

Reply:
Hi, sorry I was on holiday. Skimmed over it, looks good from afar.
We can always revisit things later if needed.